extensions/amp-story/0.1/amp-story.js

```diff
@@ -19,34 +19,734 @@
+   * @param {boolean} isEnabled
+   */
+  setAutoFullScreen(isEnabled) {
+    this.isAutoFullScreenEnabled_ = isEnabled;
+  }
+
+
+  /** @private */
+  enterFullScreen_() {
+    fullscreenEnter(this.element);
+  }
+
+
+  /**
+   * @param {boolean} opt_explicitUserAction
+   * @private
+   */
+  exitFullScreen_(opt_explicitUserAction) {
+    if (opt_explicitUserAction) {
+      this.setAutoFullScreen(false);
+    }
+
+    fullscreenExit(this.element);
+  }
+
+
+  /**
+   * Invoked when the document has actually transitioned into or out of
+   * fullscreen mode.
+   * @private
+   */
+  onFullscreenChanged_() {
+    const isFullscreen = isFullscreenElement(this.element);
+    this.systemLayer_.setInFullScreen(isFullscreen);
+  }
+
+
+  /**
+   * Shows the bookend overlay.
+   * @private
+   */
+  showBookend_() {
+    if (this.isBookendActive_) {
+      return;
+    }
+
+    this.buildBookend_().then(() => {
+      this.exitFullScreen_();
+      this.systemLayer_.toggleCloseBookendButton(true);
+      this.isBookendActive_ = true;
+
+      this.vsync_.mutate(() => {
+        this.element.classList.add('i-amphtml-story-bookend-active');
+        this.bookend_.getRoot()./*OK*/scrollTop = 0;
+      });
+    });
+  }
+
+
+  /**
+   * Hides the bookend overlay.
+   * @private
+   */
+  hideBookend_() {
+    this.systemLayer_.toggleCloseBookendButton(false);
+    this.isBookendActive_ = false;
+
+    this.vsync_.mutate(() => {
+      this.element.classList.remove('i-amphtml-story-bookend-active');
+    });
+  }
+
+
+  /**
+   * Performs a system navigation if it is determined that the specified event
+   * was a click intended for navigation.
+   * @param {!Event} event 'click' event
+   * @private
+   */
+  maybePerformSystemNavigation_(event) {
+    if (!this.isNavigationalClick_(event)) {
+      // If the system doesn't need to handle this click, then we can simply
+      // return and let the event propagate as it would have otherwise.
+      return;
+    }
+
+    event.stopPropagation();
+
+    // TODO(newmuis): This will need to be flipped for RTL.
+    const nextScreenAreaMin = this.element./*OK*/offsetLeft +
+        ((1 - NEXT_SCREEN_AREA_RATIO) * this.element./*OK*/offsetWidth);
+    const nextScreenAreaMax = this.element./*OK*/offsetLeft +
+        this.element./*OK*/offsetWidth;
+
+    if (event.pageX >= nextScreenAreaMin && event.pageX < nextScreenAreaMax) {
+      this.next_();
+    } else if (event.pageX >= this.element./*OK*/offsetLeft &&
+        event.pageX < nextScreenAreaMin) {
+      this.previous_();
+    }
+  }
+
+
+  /**
+   * @return {!Array<!Array<string>>} A 2D array representing lists of pages by
+   *     distance.  The outer array index represents the distance from the
+   *     active page; the inner array is a list of page IDs at the specified
+   *     distance.
+   */
+  getPagesByDistance_() {
+    const distanceMap = this.getPageDistanceMapHelper_(
+        /* distance */ 0, /* map */ {}, this.activePage_.element.id);
```

Comment:
First argument is priority not distance.

Reply:
The variable name was the one that was wrong